src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/CompilationContext.cs

```diff
@@ -1252,6 +1257,34 @@ private static DkmClrCompilationResultFlags GetLocalResultFlags(LocalSymbol loca
                 isIteratorOrAsyncMethod = GeneratedNames.GetKind(containingType.Name) == GeneratedNameKind.StateMachineType;
             }
 
+            var argumentNamesSet = PooledHashSet<string>.GetInstance();
```

Comment:
Please keep the original variable name `parameterNames` unless there is a reason to change it.

Reply:
OK. Let us keep parameterNames here. 

I definitely want to rename hoisted* objects because we can define those objects more precisely.

---
In reply to: [135181213](https://github.com/dotnet/roslyn/pull/21707#discussion_r135181213) [](ancestors = 135181213)